src/components/Garden/Home.js

```diff
@@ -74,6 +74,10 @@ const Home = React.memo(function Home() {
     handleShowModal('unwrap')
   }, [handleShowModal])
 
+  const handleProposalCreated = useCallback(() => {
+    filters.ranking.onChange(1)
+  }, [filters])
```

Comment:
I´m not totally convinced of this approach, plus we are kind or harcoding the "new" ranking option, but couldn´t come up with a better solution. We can consider a refactor of the filters api so that we can achieve this in a more abstracted way.